app/dashboard/views.py

```diff
@@ -240,15 +300,14 @@ def interested_profiles(request, bounty_id):
         interest_data = ProfileSerializer(interest.profile).data
         interests_data.append(interest_data)
 
-    if request.is_ajax():
-        return JsonResponse(json.dumps(interests_data), safe=False)
-
     return JsonResponse({
         'paginator': {
             'num_pages': interests.paginator.num_pages,
         },
         'data': interests_data,
-        'profile_interested': profile_interested
+        'profile_interested': profile_interested,
+        'bounty_owner_github_username': bounty.bounty_owner_github_username,
+        'can_uninterest': bounty.is_funder(handle.lower())
```

Comment:
Removed this check from front-end, if this flag is set to true, then the uninterested button will be available.